src/commands.ts

```diff
@@ -53,31 +54,57 @@ export async function executeSelection(repls: Map<string, vscode.Terminal>): Pro
     }
 }
 
-export async function compileFracasObject(filePath: string, fracasObject: string): Promise<void> {
-    const racket = getRacket();
+export function compileFracasObject(filePath: string, fracasObject: string): void {
+    const [racket, racketArgs] = getRacket();
     if (fracasObject && filePath && racket) {
         vscode.window.activeTextEditor?.document?.save();
         const cmd = `(require fracas/make-asset-json) (enter! (file "${filePath}")) (define-asset-impl: #:value ${fracasObject} #:value-name (quote ${fracasObject}) #:key (key: ${fracasObject}))`;
-        execShell(`${racket} -e "${cmd.replace(/"/g, '\\"')}"`);
+        execShell(`${racket} ${racketArgs.join(" ")} -e "${cmd.replace(/"/g, '\\"')}"`);
```

Comment:
heh how did this work before

Reply:
The collect paths used to be hard-coded in the `racketPath` config property. I just made them a separate config setting.